sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -431,6 +455,252 @@ function main() {
 }
 ```
 
+## Advanced Examples
+
+### Custom Credentials
+
+The `@azure/identity` library covers a broad range of Azure Active Directory authentication scenarios. However, it's possible the credential implementations provided might not meet the specific needs your application, or an application might want to avoid taking a dependency on the `@azure/identity` library.
+
+In this section we'll examine some example cases in which it might make sense to write a credential on your own.
+
+### Authenticating with a pre-fetched access token
+
+Our package `@azure/core-http` exports a `TokenCredential` interface which is used by the `@azure/identity` package to define a common public API for all of the Identity credentials that we offer.
```

Comment:
Early on, in JS, we made the explicit decision to house the auth related interfaces in core-auth so that folks don't have to pull in core-http. Please use `core-auth` here as well.